### PR TITLE
Updating HEART workbooks to remove private preview flag and updating overview content

### PR DESCRIPTION
**Pull request by adityaganjoo1991**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
Added 'framework by google' to the HEART Analytics - All Sections Workbook's HEART overview section. Also updated the gallery file to remove the private preview flag from all the HEART workbooks as we're ready for public preview.

HEARTworkbookchanges

### If adding or updating templates:

* [X] post a screenshot of templates and/or gallery changes
* [X] ensure your template has a corresponding gallery entry in the gallery folder
* [X] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [X] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [X] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections/HEART Analytics - All Sections.workbook [hunk 1]
```diff
@@ -848,5 +848,9 @@
       "name": "Technical Validation"
     }
   ],
+  "fallbackResourceIds": [
+    "/subscriptions/96eced6b-b9cc-4575-99ad-ff12cf25085a/resourceGroups/HEART-Scorecard-NonProd/providers/microsoft.insights/components/HeartAppInsightsDemoInstance"
+  ],
+  "fromTemplateId": "community-Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections",
```
Comment (gardnerjr) [inline Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections/HEART Analytics - All Sections.workbook:854]: the fallback resource ids and template id fields need to be removed (Build validation will catch this too)

File: gallery/workbook/microsoft.insights-components.json [hunk 2]
```diff
@@ -178,50 +178,44 @@
                     "id": "Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections",
                     "name": "HEART Analytics - All Sections",
```
Comment (gardnerjr) [inline gallery/workbook/microsoft.insights-components.json:179]: has anyone complained that these are really long names in gallery? 

they all show up as "HEART Analytics Section- ..." 

image

should that be at the end?

like "Heart Analytics - Overview",  "Adoption - HEART Analytics", "Engagement - HEART Analytics", etc, so the distinguishing thing is *first*, not at the end hidden in the overflow?

File: Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections/HEART Analytics - All Sections.workbook (hunk 1 above)
Comment (adityaganjoo1991) [reply Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections/HEART Analytics - All Sections.workbook]: Done.

**Review by gardnerjr — COMMENTED** [review]
next time you might want to revisit the names (commented above), and create an actual team for CODEOWNERS so that your "team" officially owns these files in the repo and you can approve your own things instead of it falling to the workbooks team by default.

**Review by gardnerjr — APPROVED** [review]

**Pull request merged**